lib/teleterm/api/uri/uri_test.go

```diff
@@ -43,12 +45,48 @@ func TestURI(t *testing.T) {
 		},
 	}
 
-	for _, tt := range testCases {
+	for _, tt := range tests {
+		tt := tt
 		t.Run(fmt.Sprintf("%v", tt.in), func(t *testing.T) {
+			t.Parallel()
+
 			out := tt.in.String()
-			if !reflect.DeepEqual(out, tt.out) {
-				t.Errorf("out %#v, want %#v", out, tt.out)
-			}
+			require.Equal(t, tt.out, out)
+		})
+	}
+}
+
+func TestParseClusterURI(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		in  string
+		out uri.ResourceURI
+	}{
+		{
+			"/clusters/cluster.sh",
+			uri.NewClusterURI("cluster.sh"),
+		}, {
```

Comment:
I think `gofmt` might need running on this file :)

Reply:
Hmm I'm running `gofmt` on save and running it manually `gofmt -d api/utils/uri_test.go` doesn't return anything. 🤔

Before the last commit line 64 was `}{{` instead of just `}{` and I changed it there.

Reply:
Might be weird handling of tabs by GitHub?

Reply:
Oh ok, maybe I'm confusing `gofmt` with something I've got installed, but I'd have expected line 68 to match lines 71 and 75.

Reply:
Ah, I see, that was probably a leftover of that change from `}{{` to `}{`.